src/Model/CompositeTypeGo.cs

```diff
@@ -226,35 +226,6 @@ public string Fields()
                 indented.Append(((CompositeTypeGo)BaseModelType).Fields());
             }
 
-            // If the type is a paged model type, ensure the nextLink field exists
-            // Note: Inject the field into a copy of the property list so as to not pollute the original list
-            if (!string.IsNullOrEmpty(NextLink))

```

Comment:
If NextLink is no longer required please remove it.

Reply:
It is still used on the template for paged methods convinience.